Source/Config.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace SimpleRefuel {
+	public class Config {
+		private const string _configFilePath = "GameData/SimpleRefuel/SimpleRefuel.cfg";
+		private const string SettingsNodeName = "SimpleRefuel_Settings";
+		private static Config _instance;
+
+		private static string ConfigFilePath {
+			get { return KSPUtil.ApplicationRootPath + _configFilePath; }
+		}
+
+		internal static Config Instance {
+			get { return _instance ?? ( _instance = new Config() ); }
+		}
+
+		private Config() {
+			this._load();
+		}
+
+		internal float RefuelSpeed;
+
+		private void _load() {
```

Comment:
Please do not use underscores for methods. Methods should have an uppercase first letter.